bvm/ballerina-runtime/src/main/resources/MessagesBundle.properties

```diff
@@ -196,6 +196,11 @@ merge.json.error = cannot merge JSON values of types ''{0}'' and ''{1}''
 field.removal.not.allowed = failed to remove field: ''{0}'' is a required field in ''{1}''
 operation.not.supported = {0} not supported on type ''{1}''
 invalid.fraction.digits = fraction digits cannot be less than '0'
+invalid.transaction.auto.commit.timeout = provided transaction auto commit timeout of ''{0}'' is invalid, default \
+  value of ''{1}'' will be used instead.
+invalid.transaction.cleanup.timeout = provided transaction cleanup timeout of ''{0}'' is invalid, default value of \
+  ''{1}'' will be used instead.
+
```

Comment:
Instead of defining two separate error messages, we can use a common error message similar to "Invalid transaction timeout."

Reply:
Addressed in https://github.com/ballerina-platform/ballerina-lang/pull/42940/commits/8ad8080e297833f6e923242865164c4d38fafa9d.